src/deserialize_sequence.cc

```diff
@@ -337,6 +339,33 @@ Step deserialize_step(const std::filesystem::path& path)
     return step;
 }
 
+std::istream& operator>>(std::istream& stream, Sequence& seq)
+{
+    std::string line;
+    std::string step_setup_script;
+
+    while(std::getline(stream, line, '\n'))
+        step_setup_script += (gul14::trim_right(line) + '\n');
+
+    if (not step_setup_script.empty())
+        seq.set_step_setup_script(step_setup_script);
```

Comment:
Is this what we want? Reading from an empty stream does not change the step setup script in `seq` at all? Shouldn't it rather set it to an empty string?

Reply:
It is a matter of taste. The `Sequence` is create fresh and per default it has an empty string. I can set it to the loaded step setup script independent if previously loaded or not. But what do mean with _Shouldn't it rather set it to an empty string?_?

Reply:
I mean: Assume you have a pre-existing fully filled `Sequence seq` with steps, a step setup script, and so on. Now you do `my_fstream >> seq`. If the stream is empty (say, an empty `sequence.lua`), `seq` simply keeps the old step setup script. I would consider this a bug.

We might also discuss if the `>>` operator is a good choice for this application. It makes sense for individual steps because it always reads the *entire* step, but for sequences it only imports a minor part of the entire sequence, namely the step setup script. Maybe this should rather be a normal function with a name.

Reply:
If Fini is not available at the moment, I can merge the PR. We should not forget to refactor this serialization part, though.
@MissPeppermintPatty: How do you prefer to continue?
a) Merge this PR now and open a new one for refactoring the `Sequence` serialization
b) Wait for Fini to review
c) Add the refactoring to this PR

Reply:
I would prefer point a) and open a new PR to refactor the serialization.